packages/vue/src/components/molecules/SfGallery/SfGallery.html

```diff
@@ -25,15 +24,26 @@
             class="glide__slide"
             v-for="(picture, index) in mapPictures"
             :key="'slide-' + index"
+            @mouseover="addImg(picture)"
+            @mousemove="changeImg($event,index)"
+            @mouseout="removeImg()"
           >
             <SfImage
+              ref="sfGalleryBigImage"
               class="sf-gallery__big-image"
-              :src="picture"
-              @click="$emit('click:stage', { image, index })"
+              :src="picture.normal.url"
+              @click="$emit('click:stage', { picture, index })"
+
             />
           </li>
         </ul>
       </div>
+      <div v-if="outsideZoom" ref="outSide" class="sf-gallery__zoom">
```

Comment:
- If you use JS here, maybe you can use v-if and transition to hide this element when it is not active?
![_](https://user-images.githubusercontent.com/12138170/68456580-74231a80-01fe-11ea-8837-1a89bbe6ef6c.png)
- What do you think about `position: absolute` here? And checked `.sf-gallery__stage` it have `overflow:hidden` :) 
- Maybe we can use SfImage here? @leomp12 wdyt?

Reply:
Yes, I think that if we use position:absolute for it show, we must do remove overflow: hidden of parents and happens this... show the other images of the gallery
![Captura de pantalla 2019-11-08 a las 11 09 18](https://user-images.githubusercontent.com/56393195/68469240-13094000-021a-11ea-8a86-8ef36f2d5f5b.png)

Reply:
Agree about `SfImage` and `position: absolute`, to be honest I haven't seen it was fixed positioned, I don't think it's a good ideia...
@juanfrantomaswac maybe you can put `.sf-gallery__zoom` out of stage (for outside zoom), directly child of `.sf-gallery`, wdyt?

Reply:
agree about 
> maybe you can put `.sf-gallery__zoom` out of stage (for outside zoom), directly child of `.sf-gallery`, wdyt?

Reply:
Hi  @leomp12 @przspa
I had to zoom into `.sf-gallery__stage`, to be able to put it in absolute position and that I was just below the image of `.sf-gallery__big-image`, if I put the direct son of `.sf-gallery` had problems(because it is necessary to calculate the margins of the parents of this one and I believe that it can give us many problems in a future) to put it perfectly and centered under `.sf-gallery__big-image` , what do you think?

Reply:
Hey bud, so maybe with:

```html
<div class="sf-gallery__container">
  <div class="sf-gallery__stage">
  </div>
  <div class="sf-gallery__zoom">
  </div>
</div>
```

```css
.sf-gallery__container {
  position: relative;
}
.sf-gallery__zoom {
  position: absolute;
}
```

wdyt?

Reply:
Perfect @leomp12 I try it ;)